extra_geom/base.py

```diff
@@ -311,6 +326,20 @@ def write_crystfel_geom(self, filename, *,
         """
         if nquads is None:
             nquads = self.n_quads
+
+        # If this geometry came from a .geom file, restore metadata from there
+        cfelmeta = self.metadata.get('crystfel', {})
+        if data_path is None:
+            data_path = cfelmeta.get('data') or '/entry_1/instrument_1/detector_1/data'
```

Comment:
```suggestion
            data_path = cfelmeta.get('data', default='/entry_1/instrument_1/detector_1/data')
```

Reply:
The advantage of using `or` is that it will use the default if the dict contains something like `None` - which it might, from the way we construct it (`{k: d1.get(k) for k in cfel_md_keys}`).